docs/develop/frontend-dev.md

```diff
@@ -46,18 +75,17 @@ API_BASE_URL=http://localhost:30870/api
 
     Set API_BASE_URL to provided URL instead, eg. `API_BASE_URL=http://127.0.0.1:<TUNNEL_PORT>`
 
-
-
-Start the dev server:
+Start the frontend development server:
 
 ```sh
 yarn start
 ```
 
 This will open `localhost:9870` in a new tab in your default browser.
 
-To develop against a local instance of the backend API,
-follow instructions for deploying to a local Docker instance. Update `API_BASE_URL` and then restart the dev server.
+Saving changes to files in `src` will automatically reload your browser window with the latest UI updates.
+
+To stop the development server type <kbd>Ctrl</kbd> + <kbd>C</kbd> into your command line terminal.
```

Comment:
I like using rendered keyboard keys!  We should probably enable `pymdownx.keys` in mkdocs.yml and use the plugin for this?  Enables symbols for the modifier keys which is neat.

```suggestion
To stop the development server type ++ctrl+c++ into your command line terminal.
```

Also right now the plugin isn't enabled and this suggestion won't work until it is.